_docs/how_to_run_acceptance_tests.md

```diff
@@ -0,0 +1,172 @@
+  * `BEAKER_DESTROY=no` Always leave an artifact, which likely will break future runs without cleanup.
+  * `BEAKER_DESTROY=yes` _(DEFAULT)_ always clean up.
+* `BEAKER_PROVISION` Should we ensure a clean system is built to run the suite.
+  * If `BEAKER_DESTROY` is set to no, `BEAKER_PROVISION=yes` will fail the run (because of the existing box/container)
+  * If `BEAKER_DESTROY` is set to no, `BEAKER_PROVISION=no` will run the test suite against the system anyways, which may cause environmental issues if the test suite doesn't perfectly put things to a default state.
+
+* `BEAKER_SETFILE` What should we be testing.  Beaker will call beaker-hostgenerator to create a defaultconfiguration based on this, using the known configurations for Vox Pupuli.
+* `BEAKER_PUPPET_COLLECTION` What implementation and version of puppet are we testing against
+
+
+### Listing Platforms
+
+Installed as part of the voxpupuli test suite is `metadata2gha`, which as its name suggests, is a helper to generate a json blob containing the variables for all supported oses.
+
+```shell
+# From the base directory of the module
+$ bundle exec metadata2gha | tail -n 1 | cut -d= -f2- | jq
+[
+# ... truncated for length
+  {
+    "name": "Puppet 8 - Debian 12",
+    "env": {
+      "BEAKER_PUPPET_COLLECTION": "puppet8",
+      "BEAKER_SETFILE": "debian12-64{hostname=debian12-64-puppet8}"
+    }
+  },
+  {
+    "name": "Puppet 7 - Debian 12",
+    "env": {
+      "BEAKER_PUPPET_COLLECTION": "puppet7",
+      "BEAKER_SETFILE": "debian12-64{hostname=debian12-64-puppet7}"
+    }
+  },
+# ... truncated for length
+  {
+    "name": "OpenVox 8 - Rocky 9",
+    "env": {
+      "BEAKER_PUPPET_COLLECTION": "openvox8",
+      "BEAKER_SETFILE": "rocky9-64{hostname=rocky9-64-openvox8}"
+    }
+  },
+  {
+    "name": "OpenVox 7 - Rocky 9",
+    "env": {
+      "BEAKER_PUPPET_COLLECTION": "openvox7",
+      "BEAKER_SETFILE": "rocky9-64{hostname=rocky9-64-openvox7}"
+    }
+  },
+# ... truncated for length
+]
+```
+
+This returns names that show puppet implementation (openvox or puppet) / language version, and operating system / version.
+It also provides the environment variables needed to be applied to run beaker for this combination.
+
+### Running Beaker
+
+At this point, everything is ready to run beaker.
+
+Because everything is an environment variable, there are some options on running things.
+
+1. export variables, then run `bundle exec rake beaker`
+   ```shell
+   export BEAKER_PUPPET_COLLECTION="openvox8"
+   export BEAKER_SETFILE="ubuntu2404-64{hostname=ubuntu2404-64-openvox8}"
+   bundle exec rake beaker
+   ```
+1. run `bundle exec rake beaker` with inline shell variables
+   ```shell
+   BEAKER_PUPPET_COLLECTION="openvox8" BEAKER_SETFILE="ubuntu2404-64{hostname=ubuntu2404-64-openvox8}" bundle exec rake beaker
+   ```
+
+## Adding data to a run
```

Comment:
Should this link to https://github.com/voxpupuli/voxpupuli-acceptance?tab=readme-ov-file#environment-variables-to-facts for the specific feature?